app/components/primer/icon_button.rb

```diff
@@ -18,6 +18,24 @@ class IconButton < Primer::Component
       :danger => "btn-octicon-danger"
     }.freeze
     SCHEME_OPTIONS = SCHEME_MAPPINGS.keys
+
+    # `Tooltip` that appears on mouse hover or keyboard focus over the button. Use tooltips sparingly and as a last resort.
+    # **Important:** This tooltip defaults to `type: :label`. In a few scenarios, `type: :description` may be more appropriate.
+    # Consult the <%= link_to_component(Primer::Alpha::Tooltip) %> documentation for more information.
+    #
+    # @param type [Symbol] (:description) <%= one_of(Primer::Alpha::Tooltip::TYPE_OPTIONS) %>
```

Comment:
```suggestion
    # @param type [Symbol] (:label) <%= one_of(Primer::Alpha::Tooltip::TYPE_OPTIONS) %>
```

Reply:
Actually maybe we should set a `DEFAULT_TOOLTIP_TYPE = :label` at top then set this to 
```suggestion
    # @param type [Symbol] (Primer::IconButton::DEFAULT_TOOLTIP_TYPE) <%= one_of(Primer::Alpha::Tooltip::TYPE_OPTIONS) %>
```
so we don't forget to update this if it changes in the future.